### DISCUSSION TITLE
How to remove `(major)` in PR title

### DISCUSSION THREAD

**Discussion by Xavientois** [renovatebot/renovate#12149] [Request Help] [answered]
I am aware that the commit message can be modified through the different config options (`commitMessagePrefix`, `commitMessageSuffix`, etc.), but is there a way to configure renovatebot to not append `(major)` to  the title of PRs for a major version update?

**Comment by rarkins** [comment, marked as answer]
Currently I don't think so - that logic is hardcoded. The idea is that if you have a group with both minor and major PRs at the same time then you wouldn't want them both to have the same title.
